app/utils/analytics.js

```diff
@@ -0,0 +1,3 @@
+import firebase from 'react-native-firebase';
```

Comment:
Analytics is a type of log, so I think we could add this to `utils/log`.
We could even have a `logEvent` that sents data both to Crashlytics and Bugsnag.
That would help to analyze how our users are using the app and to track errors.